### ISSUE TITLE
Can't parse description field in asset registration

### ISSUE THREAD

**Issue by MarcusJones** [oceanprotocol/plecos#25]
## Current Behavior
The following description will cause errors and break the registration flow;
```
Line 1

Line 2
```
Error trace (but just see for yourself)
```
Response message: 
[{"message":"'Line 1\\n\\nLine 2' does not match '^(.*)$'","path":"base/description"}]

Error fetching querying metadata:  
Response {type: "cors", url: "https://aquarius.commons.oceanprotocol.com/api/v1/aquarius/assets/ddo", redirected: false, status: 400, ok: false, …}
Logger.js:101 

DDO stored

Logger.js:101 error: Cannot read property 'id' of null
```

This occurs in BMW, Commons, Nile, ...!

### Possible Solution

Plecos? Try the /validate endpoint in Aquarius service?

**Comment by kremalicious** [comment]
No idea if that means something but what's noticeable is that from Commons the request payload includes the `\n` as expected:

<img width="433" alt="Screen Shot 2019-11-06 at 10 50 51" src="https://user-images.githubusercontent.com/90316/68287671-64cb9200-0083-11ea-8b75-46c5bab9aec9.png">

But the plecos error message response has them escaped:

<img width="672" alt="Screen Shot 2019-11-06 at 10 55 29" src="https://user-images.githubusercontent.com/90316/68288027-fc30e500-0083-11ea-965b-234c6273f18b.png">

Which makes sense because in JSON a `\n` will create a newline in the actual JSON source. So I think we might need to escape newline characters before sending into Aquarius, but then they would need to be unescaped before being displayed when retrieved.

**Comment by kremalicious** [comment]
I have a workaround in Commons which is just a string replacement:

```js
description.replace(/\n/g, '\\n')
```

But maybe Plecos should actually handle that and escape all `\n` into `\\n`?

**Comment by alexcos20** [comment]
I agree, it should be handled inside Plecos

**Comment by 0x3bfc** [comment]
@alexcos20 @MarcusJones Is it fixed ?

**Comment by ssallam** [comment]
This is now fixed in plecos v0.7.10 and aquarius v0.3.9. Please verify.

**Comment by kremalicious** [comment]
Updated all Aquarius instances in all networks to `v0.3.9`. Verified it's working, was able to publish with newlines: https://commons.nile.dev-ocean.com/asset/did:op:99380829d1ae47ffb1dc322254dfe6399890861c4a544df38524f94a4f84b22d

Thanks everyone!
